app/src/main/java/com/github/gotify/NotificationSupport.java

```diff
@@ -32,19 +36,69 @@ public static void createChannels(NotificationManager notificationManager) {
                             Channel.FOREGROUND,
                             "Gotify foreground notification",
                             NotificationManager.IMPORTANCE_LOW);
-            // High importance for message notifications so that they are shown as heads-up
-            // notifications and sorted towards the top of the notification shade
-            NotificationChannel messages =
+
+            NotificationChannel messagesImportanceMin =
+                    new NotificationChannel(
+                            Channel.MESSAGES_IMPORTANCE_MIN,
+                            "Min importance Gotify messages",
```

Comment:
How about specifying what priority is meant with "min", "default", ..,  so maybe 
`min priority messages (<1)`, `low priority messages (1-3)`, `normal priority messages (4-7)` and `high priority messages (>7)`

Reply:
Good idea! Otherwise its not immediately obvious which channel maps to which priorities. I just made the change!